shotover-proxy/Cargo.toml

```diff
@@ -63,7 +63,7 @@ csv = "1.1.6"
 strum_macros = "0.23"
 
 [dev-dependencies]
-criterion = { version = "0.3", features = ["async_tokio", "html_reports"] }
+criterion = { git = "https://github.com/bheisler/criterion.rs", branch = "version-0.4", version = "0.3", features = ["async_tokio", "html_reports"] }
```

Comment:
Is it worth just waiting for the crate to get published or is this blocking something urgent?

Reply:
We can work around the issue by temporarily disabling merge protection and then merging while CI is red.
Not a great habit to form though.

I'm not currently writing a benchmark writing benchmarks is something I expect people to be doing all the time.

Reply:
Happy to just use Git for now. I'll watch [this PR](https://github.com/bheisler/criterion.rs/pull/496) so we can upgrade once it's released.